server/public/reset.css

```diff
@@ -3,5 +3,21 @@
 }
 
 body {
+  background: #fafafa;
+  display: flex;
+  justify-content: center;
+  min-height: 100vh;
```

Comment:
Bootstrap has helper classes for flex, background, min height, etc

https://getbootstrap.com/docs/5.0/utilities/background/